File: Python/1_Iniciante/1066.py
pares = 0
impares = 0
positivos = 0
negativos = 0

for i in range (5):
    a = float(input())
    if(a%2==0):
        pares+=1
    if(a%2!=0):
        impares+=1
    if(a>0):
        positivos+=1
    if(a<0):
        negativos+=1

print(f'{pares} valor(es) par(es)')
print(f'{impares} valor(es) impar(es)')
print(f'{positivos} valor(es) positivo(s)')
print(f'{negativos} valor(es) negativo(s)')

File: Ruby/1_Iniciante/1066.rb
pares = 0
impares = 0
positivos = 0
negativos = 0

5.times do |i|
    a = gets.to_f
    if a%2 == 0
        pares+=1
    end
    if a%2 !=0
        impares+=1
    end
    if a > 0
        positivos+=1
    end
    if a < 0
        negativos+=1
    end
end

puts "#{pares} valor(es) par(es)"
puts "#{impares} valor(es) impar(es)"
puts "#{positivos} valor(es) positivo(s)"
puts "#{negativos} valor(es) negativo(s)"
